README.md

```diff
@@ -80,7 +80,7 @@ notification = Grocer::Notification.new(
   content_available: true                  # optional; any truthy value will set 'content-available' to 1
 )
 
-pusher.push(notification)
+pusher.push(notification) # return value is the number of bytes sent successfully
```

Comment:
Good idea. What if instead of a comment we assigned the returned value to a local variable. Named something like `bytes_sent` or `number_of_bytes_sent`?